BackofficeBundle/EventSubscriber/DataTransformer/ValueTransformerInterface.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace OpenOrchestra\BackofficeBundle\EventSubscriber\DataTransformer;
+
+/**
+ * ValueTransformerInterface
+ */
+interface ValueTransformerInterface
```

Comment:
there should be a support method.

The name can be too vague to decide which transformer you will use